### DISCUSSION TITLE
Matter Manipulator upgrades

### DISCUSSION THREAD

**Discussion by Rirbe** [wb1016/sb_korpatch#45] [unfinished]
Matter Manipulator upgrades => 물질 조작기 업그레이드
Manipulate what matters on your matter manipulator!
여러분의 물질 조작기에 중요한 것을 조작하세요!

MATTER PROC UNIT
물질 조작 장치
Increase area of effect to 3x3 tiles
효과 영역을 3x3 타일로 늘립니다.
Increase area of effect to 4x4 tiles
효과 영역을 4x4 타일로 늘립니다.
Increase area of effect to 5x5 tiles
효과 영역을 5x5 타일로 늘립니다.

EXPANSION SLOT
확장 슬롯
Enable the collection of liquids using Excavation Mode
액체 수집을 위한 확장 굴착 모드 활성화
Enable Paint Mode, for painting tiles in a variety of colours
다양한 색상으로 타일을 칠하기 위해 페인트 모드 활성화
Enable Wire Mode, for wiring together functional objects
기능성 물체를 함께 배선하기 위한 배선 모드 활성화

POWER GENERATOR
발전 장치
Increase matter deconstruction rate by 100%
물질 해체율 100% 향상
Increase matter deconstruction rate by 200%
물질 해체율 200% 향상
Increase matter deconstruction rate by 300%
물질 해체율 300% 향상

OPTICS
광학 장치
Increase effective manipulator range by 2 tiles
조작기 효과 범위 2타일 증가
Increase effective manipulator range by 4 tilesMatter Manipulator upgrades => 물질 조작기 업그레이드
Manipulate what matters on your matter manipulator!
여러분의 물질 조작기에 중요한 것을 조작하세요!

MATTER PROC UNIT
물질 조작 장치
Increase area of effect to 3x3 tiles
효과 영역을 3x3 타일로 늘립니다.
Increase area of effect to 4x4 tiles
효과 영역을 4x4 타일로 늘립니다.
Increase area of effect to 5x5 tiles
효과 영역을 5x5 타일로 늘립니다.

EXPANSION SLOT
확장 슬롯
Enable the collection of liquids using Excavation Mode
액체 수집을 위한 확장 굴착 모드 활성화
Enable Paint Mode, for painting tiles in a variety of colours
다양한 색상으로 타일을 칠하기 위해 페인트 모드 활성화
Enable Wire Mode, for wiring together functional objects
기능성 물체를 함께 배선하기 위한 배선 모드 활성화

POWER GENERATOR
발전 장치
Increase matter deconstruction rate by 100%
물질 해체율 100% 향상
Increase matter deconstruction rate by 200%
물질 해체율 200% 향상
Increase matter deconstruction rate by 300%
물질 해체율 300% 향상

OPTICS
광학 장치
Increase effective manipulator range by 2 tiles
조작기 효과 범위 2타일 증가
Increase effective manipulator range by 4 tiles
조작기 효과 범위 4타일 증가
Increase effective manipulator range by 6 tiles
조작기 효과 범위 6타일 증가

**Comment by wb1016** [comment]
여긴 이미 제가 작업을 끝냈습니다. 코드를 적용할 시 게임이 크래시떠서 안 넣는 것 뿐입니다.
